<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Petition">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:purple; 
color: pink ;/* rgb(255, 0, 255);*/
border-style:outset ; 
border-width:3px ; 
border-radius:10px;

   
</string>
    </property>
    <property name="text">
     <string>Petition</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Materiel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:purple; 
color: pink ;/* rgb(255, 0, 255);*/
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Materiel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Agent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>212</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:purple; 
color: pink ;/* rgb(255, 0, 255);*/
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Agent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Crimes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>262</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:purple; 
color: pink ;/* rgb(255, 0, 255);*/
border-style:outset ; 
border-width:3px ; 
border-radius:10px;

</string>
    </property>
    <property name="text">
     <string>Crimes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plaintes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:purple; 
color: pink ;/* rgb(255, 0, 255);*/
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Plaintes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mission">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:purple; 
color: pink ;/* rgb(255, 0, 255);*/
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Mission</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>631</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_4"/>
    <widget class="QWidget" name="page_5"/>
    <widget class="QWidget" name="page_6"/>
    <widget class="QWidget" name="page_7"/>
    <widget class="QWidget" name="page_2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>10</y>
        <width>681</width>
        <height>541</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="mission">
       <attribute name="title">
        <string>VEHICULE</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>-20</x>
          <y>0</y>
          <width>771</width>
          <height>491</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="pb_ajouter">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>180</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>40</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Id</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>90</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Marque</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>140</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Mod√©le</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_supprimer">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>80</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_nom">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>90</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_prenom">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>140</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QTableView" name="tab_vehicule">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>270</y>
           <width>321</width>
           <height>192</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>40</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Identifiant</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id_supp">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="modifier_vehicule">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>230</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>modifier</string>
         </property>
        </widget>
        <widget class="QLabel" name="text">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>0</y>
           <width>186</width>
           <height>22</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string> Smart  Police  Station</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>mission</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>0</y>
          <width>771</width>
          <height>491</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="pb_ajouter_2">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>100</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLabel" name="l1">
         <property name="geometry">
          <rect>
           <x>86</x>
           <y>60</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string> ID mission</string>
         </property>
        </widget>
        <widget class="QLabel" name="l2">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>90</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>nbr vehicule</string>
         </property>
        </widget>
        <widget class="QLabel" name="l3">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>120</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>nbr policier</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_supprimer_2">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>80</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>60</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_nom_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>90</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_prenom_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>120</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QTableView" name="tab_mission">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>270</y>
           <width>411</width>
           <height>192</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="l4_2">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>40</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Identifiant</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id_supp_2">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="l4">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>150</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Date mission</string>
         </property>
        </widget>
        <widget class="QPushButton" name="modifier_mission">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>280</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
        <widget class="QPushButton" name="tri_mission">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>240</y>
           <width>41</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>Tri </string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_mission">
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>240</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>id</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>nbr de vehicule</string>
          </property>
         </item>
        </widget>
        <widget class="QLineEdit" name="le_nom_3">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>150</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>240</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;
</string>
         </property>
         <property name="text">
          <string>PDF</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="recher_mission">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>240</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_rechercher_mission">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>240</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>rechercher</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>580</x>
           <y>120</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Tab mis_veh</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>360</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>0</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>FR</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>0</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ENG</string>
         </property>
        </widget>
        <widget class="QLabel" name="policou_9">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>0</y>
           <width>191</width>
           <height>35</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string> Smart  Police  Station</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>150</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>Envoyer</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-1</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color :transparent;
border-image:url(C:/Users/Utilisateur/Downloads/ifinal/Logo.png);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
